src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -352,6 +352,15 @@ void S6Exporter::Export()
     _s6.park_rating = gParkRating;
 
     std::memcpy(_s6.park_rating_history, gParkRatingHistory, sizeof(_s6.park_rating_history));
```

Comment:
Also this line would be better as
```suggestion
        std::copy(std::begin(gParkRatingHistory), std::end(gParkRatingHistory), _s6.park_rating_history);
```
If someone changes with the type this will no longer compile making it safe.